conf.py

```diff
@@ -134,8 +134,8 @@
 # smv_tag_whitelist = None
 
 smv_branch_whitelist = r"^(main|humble)$"
-smv_released_pattern = r"^refs/(heads|remotes/[^/]+)/(humble).*$"
-smv_latest_version = "humble"
+smv_released_pattern = r"^refs/(heads|remotes/[^/]+)/(main|humble).*$"
+smv_latest_version = "main"
```

Comment:
I've been thinking for a while now that "main" is an awkward directory for docs. How about we switch to having a "latest" branch for tutorials which would be more intuitive and consistent with readthedocs, the old ROS API, and plenty other docs out there.

Reply:
We'd have to change the branch to do that and it would affect the URL. I like main because it is the name of the branch in MoveIt we also use.